### PR TITLE
STREAMP-2259 kafkaEventReceiver bug fix

### PR DESCRIPTION
**Pull request by a-nersharma**
# stream-registry PR

_&lt;High level description of the PR&gt;_

### Changed
* bugfix on KafkaEventReceiver

# PR Checklist Forms

- [x] CHANGELOG.md updated
- [x] Reviewer assigned
- [ ] PR assigned (presumably to submitter)
- [ ] Labels added (enhancement, bug, documentation)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## unreleased - 2020-06-23
```
Comment (nahguam) [inline CHANGELOG.md:7]:
```suggestion
## [Unreleased]
```

File: state/kafka-receiver/src/main/java/com/expediagroup/streamplatform/streamregistry/state/kafka/KafkaEventReceiver.java [hunk 2]
```diff
@@ -123,7 +123,11 @@ void consume(EventReceiverListener listener) {
       for (ConsumerRecord<AvroKey, AvroValue> record : consumer.poll(Duration.ofMillis(100))) {
         var event = converter.toModel(record.key(), record.value());
         currentOffset.set(record.offset());
-        listener.onEvent(event);
+        try {
+          listener.onEvent(event);
+        } catch (Exception e) {
+          log.error("listener.onEvent failed", e);
```
Comment (nahguam) [inline state/kafka-receiver/src/main/java/com/expediagroup/streamplatform/streamregistry/state/kafka/KafkaEventReceiver.java:129]:
```suggestion
          log.error("Listener failed for event {}", event, e);
```

File: CHANGELOG.md (hunk 1 above)
Comment (a-nersharma) [reply CHANGELOG.md]: done

File: state/kafka-receiver/src/main/java/com/expediagroup/streamplatform/streamregistry/state/kafka/KafkaEventReceiver.java (hunk 2 above)
Comment (a-nersharma) [reply state/kafka-receiver/src/main/java/com/expediagroup/streamplatform/streamregistry/state/kafka/KafkaEventReceiver.java]: done

**Pull request merged**